### ISSUE TITLE
Add a way to disable ReCAPTCHA during development

### ISSUE THREAD

**Issue by Minoru** [codingteam/loglist#149]
It'd be nice to flip a toggle or something and be able to develop locally without worrying about ReCAPTCHA. As it is, I had to edit a couple of files to make Loglist believe that CAPTCHA check was passed successfully.

**Comment by rexim** [comment]
This is strange. ReCAPTCHA should work if your host is localhost.

**Comment by ForNeVeR** [comment]
Maybe he wanted to edit the files without an active internet connection? I think that's a valid point.

**Comment by rexim** [comment]
@Minoru did you, really?

**Comment by rexim** [comment]
Anyway, we have config for such thing. Feel free to add an option.

**Comment by Minoru** [comment]
I have an active Internet connection. No, I didn't try to work on LogList without one. Still, ReCAPTCHA doesn't work, claiming I have invalid site key. [The docs](https://developers.google.com/recaptcha/docs/start) claim that "[b]y default, all keys work on "localhost" (or "127.0.0.1"), so you can always develop and test on your local machine.", but that doesn't seem to be the case. Any advice?

**Comment by Minoru** [comment]
JFTR, I didn't edit RECAPTCHA_{PUBLIC,PRIVATE}_KEY settings in `devenv` file. Should I?

**Comment by rexim** [comment]
Hah! Looks like Mr @ForNeVeR and I use the the same key that are in production! :D

**Comment by rexim** [comment]
That's why everything work for us.

**Comment by rexim** [comment]
You can create your own RECAPTCHA keys if you want

**Comment by Minoru** [comment]
Yeah, so it turns out I should generate and specify valid keys in order for ReCAPTCHA to work. I'll update the README in the moment.

**Comment by ForNeVeR** [comment]
It seems that you need to obtain your own key even to use it on localhost. I think that we should add an option to disable CAPTCHA. It will be easier for our contributors to install local instances without registration and SMS, you know.

**Comment by Minoru** [comment]
I don't see a clear way to add an option, so the only thing I can do right now is clarify the docs. Will such a PR be welcome?

**Comment by ForNeVeR** [comment]
Yes, it will be good (but not enough). I guess someone else could later resolve the issue completely.

**Comment by rexim** [comment]
K, taking this one then

**Comment by rexim** [comment]
> partially fixes

**Comment by ForNeVeR** [comment]
@Minoru, you should definitely be more careful with your words.

**Comment by Aakash2408** [comment]
can i block Recaptcha on specific dev environments in my codebase?

**Comment by Minoru** [comment]
@Aakash2408 Currently you can't, we haven't implemented a configuration setting for that yet. Would you like to work on that?
